.github/workflows/sonar-scan.yml

```diff
@@ -101,7 +101,7 @@ jobs:
         run: sed -i 's+/home/runner/work/feeds-manager/feeds-manager/+/github/workspace/+g' ${{ steps.sonarqube_report_paths.outputs.sonarqube_eslint_report_paths }}
 
       - name: SonarQube Scan
-        uses: sonarsource/sonarqube-scan-action@a6ba0aafc293e03de5437af7edbc97f7d3ebc91a # v1.2.0
+        uses: sonarsource/sonarqube-scan-action@2.0.1
```

Comment:
I think for security reasons we're always supposed to pin to a sha rather than a tag